src/core_plugins/timelion/server/series_functions/es/index.js

```diff
@@ -27,7 +27,7 @@ export default new Datasource('es', {
     {
       name: 'index',
       types: ['string', 'null'],
-      help: 'Index to query, wildcards accepted'
+      help: 'Index to query, wildcards accepted. Provide Index Pattern name for scripted field support.'
```

Comment:
This is somewhat of odd, and I might be missing something.

Have you tested this with the default `logstash-*` index-pattern, from the examples with makelogs. I tried it, and no matching saved-objects come up. Creating a new index pattern, something like `log*`, does make it work.